modules/budget/variables.tf

```diff
@@ -70,3 +70,13 @@ variable "monitoring_notification_channels" {
   type        = list(string)
   default     = []
 }
+
+variable "labels" {
```

Comment:
Can we test this by modifying this example to add a label
https://github.com/terraform-google-modules/terraform-google-project-factory/blob/e444e2ae47632bfc7f1c060be6db1ab15e1cfb9d/examples/budget_project/main.tf#L40

Reply:
Sure. I will probably do that as a separate commit (rather than squashing all together)